src/screens/Booking/components/PassengerForm/Form.tsx

```diff
@@ -0,0 +1,101 @@
+import {useFormik} from 'formik';
+import React from 'react';
+import * as Yup from 'yup';
+import {t} from '@translations';
+import {useDispatch} from 'react-redux';
+import {AppDispatch} from '@store/store';
+import {buy} from '@modules/ticket';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {InputField} from '@components';
+import styles from './Form.styles';
+
+interface FormProps {
+  flightId: string;
+}
+
+const Form = ({flightId}: FormProps) => {
+  const initialValues = {
+    name: '',
+    surname: '',
+    passport: '',
```

Comment:
this const definition (initialization) can be done outside the Form component as it's not dependent on the Form component